packages/flame-css/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@lightspeed/flame-css",
+  "version": "0.0.0",
+  "private": true,
```

Comment:
FYI I'd start with `0.1.0` when we `yarn release`. See https://github.com/lightspeed/flame/blob/next/RELEASE.md#note-on-new-packages.